README.md

```diff
@@ -155,9 +155,9 @@ val deliveryStream: Resource[Task, fs2.Stream[Task, StreamedResult]] = for {
     val stream: fs2.Stream[Task, StreamedResult] = streamingConsumer.deliveryStream.through(processMyStream)
     
     // create resilient (self-restarting) stream; see more information below
-    val resilientStream: fs2.Stream[Task, StreamedResult] = stream.handleErrorWith { _ =>
+    val resilientStream: fs2.Stream[Task, StreamedResult] = stream.handleErrorWith { e =>
```

Comment:
why not keep `_`?

Reply:
To show that there is an exception that should be processed somehow.

Reply:
While I think it's obvious the error is there, I have no problem with this.